[2022-10-23 21:12:40,838] INFO [GroupCoordinator 1]: Member consumer-test-user-1-f4fb9739-2246-4ae2-9e8c-24bac439d991 in group test-user has failed, removing it from the group (kafka.coordinator.group.GroupCoordinator)
[2022-10-23 21:12:40,856] INFO [GroupCoordinator 1]: Preparing to rebalance group test-user in state PreparingRebalance with old generation 1 (__consumer_offsets-22) (reason: removing member consumer-test-user-1-f4fb9739-2246-4ae2-9e8c-24bac439d991 on heartbeat expiration) (kafka.coordinator.group.GroupCoordinator)
[2022-10-23 21:12:40,859] INFO [GroupCoordinator 1]: Group test-user with generation 2 is now empty (__consumer_offsets-22) (kafka.coordinator.group.GroupCoordinator)
[2022-10-23 21:12:44,524] INFO Terminating process due to signal SIGTERM (org.apache.kafka.common.utils.LoggingSignalHandler)
[2022-10-23 21:12:44,548] INFO [KafkaServer id=1] shutting down (kafka.server.KafkaServer)
[2022-10-23 21:12:44,558] INFO [KafkaServer id=1] Starting controlled shutdown (kafka.server.KafkaServer)
[2022-10-23 21:12:45,074] WARN Session 0x10000032f480001 for sever localhost/127.0.0.1:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
java.net.SocketException: Connection reset
	at java.base/sun.nio.ch.SocketChannelImpl.throwConnectionReset(SocketChannelImpl.java:394)
	at java.base/sun.nio.ch.SocketChannelImpl.read(SocketChannelImpl.java:426)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doIO(ClientCnxnSocketNIO.java:74)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:350)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1290)
[2022-10-23 21:12:47,218] INFO Opening socket connection to server localhost/[0:0:0:0:0:0:0:1]:2181. (org.apache.zookeeper.ClientCnxn)
[2022-10-23 21:12:47,254] WARN Session 0x10000032f480001 for sever localhost/[0:0:0:0:0:0:0:1]:2181, Closing socket connection. Attempting reconnect except it is a SessionExpiredException. (org.apache.zookeeper.ClientCnxn)
java.net.ConnectException: Connection refused
	at java.base/sun.nio.ch.Net.pollConnect(Native Method)
	at java.base/sun.nio.ch.Net.pollConnectNow(Net.java:672)
	at java.base/sun.nio.ch.SocketChannelImpl.finishConnect(SocketChannelImpl.java:946)
	at org.apache.zookeeper.ClientCnxnSocketNIO.doTransport(ClientCnxnSocketNIO.java:344)
	at org.apache.zookeeper.ClientCnxn$SendThread.run(ClientCnxn.java:1290)
